protocol_tests/connection/__init__.py

```diff
@@ -337,3 +338,153 @@ def verify_sig(self, expected_signer: str):
     def get_connection_info(self):
         """Get connection information out of Request Message."""
         return DIDDoc(self['connection']['DIDDoc']).get_connection_info()
+
+
+class Ack(Message):
```

Comment:
I'd like to see code for messages not strictly related to the connection protocol placed in a more appropriate location, such as `protocol_tests/trust_ping` for the `Ping` classes.